packages/manager/apps/web/client/app/components/hosting-domain-offers/constants.js

```diff
@@ -0,0 +1,126 @@
+import moment from 'moment';
+
+export const UCENTS_FACTOR = 1000000000;
+
+export const NEW_OFFERS_START_DATE = '2023-06-15';
+
+export const NEW_OFFERS_END_DATE = moment(NEW_OFFERS_START_DATE).add(6, 'M');
```

Comment:
It's a bit confusing to have start date and end date with a different type (string vs date)

Reply:
but right could be improved bit.
`moment('2023-06-15')`